client/src/components/layout/ReviewTile.js

```diff
@@ -1,10 +1,10 @@
 import React from "react"
 
-const ReviewTile = ({ id, rating, comment }) => {
+const ReviewTile = ({review}) => {
```

Comment:
space padding inside curly braces
https://github.com/airbnb/javascript?tab=readme-ov-file#whitespace--in-braces